src/LogWrapper.ts

```diff
@@ -29,6 +29,12 @@ interface HookshotLogInfo extends winston.Logform.TransformableInfo {
 }
 export default class LogWrapper {
 
+    static isConfigured: boolean;
```

Comment:
Should this be `private`, since it has a getter?

Reply:
Yes -- this is not rust